crates/subspace-runtime/src/lib.rs

```diff
@@ -570,9 +570,8 @@ parameter_types! {
     pub const MaxBundlesPerBlock: u32 = 10;
     pub const DomainInstantiationDeposit: Balance = 100 * SSC;
     pub const MaxDomainNameLength: u32 = 32;
-    pub const BlockTreePruningDepth: u32 = 256;
-    // TODO: revisit these
-    pub const StakeWithdrawalLockingPeriod: DomainNumber = 256;
+    pub const BlockTreePruningDepth: u32 = 14_400;
```

Comment:
Isn't it a problem that so much state is being kept for every domain though?

Reply:
That is fair. We would have to see the actual state stored during gemini-3h. In the meantime, we can already start looking at removing state we don't need during challenge period but I would not have too high hopes on this yet

Reply:
Isn't it deterministic how much state it will occupy?